tsqa/environment.py

```diff
@@ -315,6 +317,21 @@ def __init__(self, layout=None):
         else:
             self.layout = None
 
+        #process environment options
+        self.keep_tmp = False
+        if conf_plugin.args.keep_tmp:
+            self.keep_tmp=True
+        self.sleep_in_sec = 0
+        if conf_plugin.args.sleep_in_sec:
+            self.sleep_in_sec = conf_plugin.args.sleep_in_sec
+        self.standalone_ats_port = -1
```

Comment:
As mentioned above, this will need to not only change this-- but we'll need to coordinate with the test cases (through the env class) what the capabilities of this "standalone" ats are. This will at least include what that socket is capable of, but to match completely with the rest of the tests this would require us parsing the config files used by that standalone ATS (so tests that run conditionally based on certain options can do so).